great_expectations/datasource/data_connector/util.py

```diff
@@ -515,3 +518,24 @@ def _build_sorter_from_config(sorter_config: Dict[str, Any]) -> Sorter:
         },
     )
     return sorter
+
+
+def _build_asset_from_config(
+    runtime_environment: "DataConnector", config: dict
+) -> Asset:
+    """Build Asset from configuration and return asset. Used by both ConfiguredAssetDataConnector and RuntimeDataConnector"""
+    runtime_environment: dict = {"data_connector": runtime_environment}
+    config = assetConfigSchema.load(config)
+    config = assetConfigSchema.dump(config)
+    asset: Asset = instantiate_class_from_config(
+        config=config,
+        runtime_environment=runtime_environment,
+        config_defaults={},
+    )
+    if not asset:
+        raise ge_exceptions.ClassInstantiationError(
+            module_name="great_expectations.datasource.data_connector.asset",
+            package_name=None,
+            class_name=config["class_name"],
+        )
+    return asset
```

Comment:
Shared by `configured_asset_file_path_data_connector.py` and `runtime_data_connector.py`. Moved to a `util.py` file.